README.md

```diff
@@ -705,9 +705,9 @@ access to the corpora domain.
 
 These need to be set as environment variables as follows:
 
-```
-export HQ_USERNAME=<username>
-export HQ_API_KEY=<apikey>
+```shell
+$ export HQ_USERNAME=<username>
+$ export HQ_API_KEY=<apikey>
```

Comment:
nit:
Should not we generally avoid "$"?
Whenever I need to simply copy the command and run it, I need to remove that $ sign each time after pasting that into terminal.

Reply:
I added them in here to be consistent with the rest of the document (although I've just noticed I missed two others).

Generally, I have found it useful to indicate the prompt to make it clear what kind of prompt it is. e.g.
```
$ psql -U commcarehq

=# \l
    List of databases
...
=# \q

$ python3

>>> print('Hello world.')
Hello world.
```

I have found that often writers will leave out the prompt so consensus is not universal, but that more often than not, they include it. Certainly across Dimagi docs, we seem to include it most of the time.